# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/2.2.2-test-lfs-timeout.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


set the timeout for Reiserfs to 120 seconds

--- ./io/test-lfs.c~	Fri Feb  9 18:04:07 2001
+++ ./io/test-lfs.c	Sat Feb 17 04:30:18 2001
@@ -34,7 +34,7 @@
 #define PREPARE do_prepare
 
 /* We might need a bit longer timeout.  */
-#define TIMEOUT 20 /* sec */
+#define TIMEOUT 120 /* sec */
 
 /* This defines the `main' function and some more.  */
 #include <test-skeleton.c>
